docs/guide/cleanup.md

```diff
@@ -82,6 +82,9 @@ Even if you will try tackle them yourself, this will help improve visibility so
 
 Or if you do not have time to tackle it yourself, it is possible someone else can follow your findings.
 
+Feel free to also attach a package report using [DevMiner's package-size-calculator](https://github.com/TheDevMinerTV/package-size-calculator) to the issue, to give an overview over the package's size and the changes you are proposing.
+![Package Report](../public/pkgcli-size-report.png)
```

Comment:
Nice! I think we should leave only the paragraph. If we'd like to add the image, we should have it as a code block but probably in this case it would be a bit much for the current state of the guide. You can also include this tool in the Resources guide.

Reply:
I've removed the screenshot and added the tool to the resources page as well.